### PR TITLE
feat(cb2-7789): create functionality to remove a test from cert gen

### PR DESCRIPTION
**Pull request by naathanbrown**
## Ticket title

We need to remove broken certs that have now been cancelled.
[https://dvsa.atlassian.net/browse/CB2-7789](https://github.com/dvsa/cvs-tsk-cert-gen-init/pull/7789)

## Checklist

- [ ] Branch is rebased against the latest develop/common
- [ ] Necessary `id` required prepended with `"test-"` have been checked with automation testers and added
- [ ] Code and UI has been tested manually after the additional changes
- [ ] PR title includes the JIRA ticket number
- [ ] Squashed commits contain the JIRA ticket number
- [ ] Link to the PR added to the repo
- [ ] Delete branch after merge

### CODE REVIEW

**Review by leifkemp-bjss — DISMISSED** [review]

File: src/functions/certGen.ts [hunk 1]
```diff
@@ -35,11 +35,17 @@ const certGen: Handler = async (
     );
   const certificateUploadService: CertificateUploadService =
     Injector.resolve<CertificateUploadService>(CertificateUploadService);
-  const certificateUploadPromises: Array<Promise<ManagedUpload.SendData>> = [];
+  const certificateUploadPromises: Array<
+    Promise<ManagedUpload.SendData> | Promise<any>
```
Comment (gjulien-bjss) [inline src/functions/certGen.ts:39]:
```suggestion
    Promise<ManagedUpload.SendData> | Promise<ManagedUpload.DeleteData>
```
If that's a thing ?

Comment (naathanbrown) [reply src/functions/certGen.ts:39]: Who knows, dare you to find anything on AWS documentation.

Comment (naathanbrown) [reply src/functions/certGen.ts:39]: In a serious answer, it's this `PromiseResult<S3.DeleteObjectOutput, AWSError>` but I just cannot make it play ball, and for the sake of some better typing on something that is awful anyway, what's the point in sinking hours into this.

Comment (naathanbrown) [reply src/functions/certGen.ts:39]: I got angry and found a solution, what do you think of that change?

**Review by gjulien-bjss — APPROVED** [review]

**Review by leifkemp-bjss — APPROVED** [review]

**Pull request merged**